### ISSUE TITLE
End of package body not detected with Oracle

### ISSUE THREAD

**Issue by laurentgrangier** [flyway/flyway#2783]
###### Which version and edition of Flyway are you using?

Flyway 6.4.0 (currently the latest)

###### Which client are you using? (Command-line, Java API, Maven plugin, Gradle plugin)

Maven Plugin

###### Which database are you using (type & version)?

Oracle 18.0

###### Which operating system are you using?

Windows

###### What did you do?
(Please include the content causing the issue, any relevant configuration settings, the SQL statement that failed (if relevant) and the command you ran.)

Try to run the following migration script:

```
CREATE OR REPLACE PACKAGE my_package AS
    PROCEDURE my_procedure (a IN VARCHAR2 DEFAULT NULL);
END;
/

CREATE OR REPLACE PACKAGE BODY my_package AS

  PROCEDURE my_procedure (a IN VARCHAR2 DEFAULT NULL) IS
  BEGIN
    dbms_output.put_line(a);
  END;

BEGIN
    my_procedure('test');
END;
/


CREATE TABLE my_table (
    my_column NUMBER(1)
);
```

###### What did you expect to see?

I expect to see no error and the package "my_package" and the table "my_table" created.

###### What did you see instead?

The following error:

`[WARNING] DB: Warning: Ausführung mit Warnung abgeschlossen (SQL State: 99999 - Error Code: 17110)`

And the table `my_table` is not created.

Using the debug mode, I see that the end of the package body is not recognized and Flyway try to execute the `create package body` and `create table` together as a single statement.

If I run the migration script using Toad, there is no error and everything is created correctly.

**Comment by Lyeeedar** [comment]
Would you be able to test if this happens on flyway 6.3.2?

**Comment by laurentgrangier** [comment]
@Lyeeedar I tested with Flyway 6.3.2 and it doesn't work either. I also tested unsuccessfuly with an older version (5.2.4).

**Comment by Lyeeedar** [comment]
Thank you for trying with some older version, at least I know now its not a regression. I shall investigate further.

**Comment by Lyeeedar** [comment]
This should hopefully be fixed by this commit https://github.com/flyway/flyway/commit/a29024b818f2bee728d192dcbbf65f406a219cfe and released in Flyway 6.4.1 (sometime next week)

**Comment by laurentgrangier** [comment]
Thank you 👍
